pkg/actors/actors.go

```diff
@@ -106,6 +106,9 @@ type Actors interface {
 	CreateTimer(ctx context.Context, req *CreateTimerRequest) error
 	// DeleteTimer deletes an actor timer.
 	DeleteTimer(ctx context.Context, req *DeleteTimerRequest) error
+	IsActorHosted(ctx context.Context, req *ActorHostedRequest) bool
+	GetActiveActorsCount(ctx context.Context) []*runtimev1pb.ActiveActorsCount
+	RegisterInternalActor(ctx context.Context, actorType string, actor InternalActor, actorIdleTimeout time.Duration) error
```

Comment:
Remove - they are now in the ActorRuntime interface (see #6633 )

```suggestion
```

Reply:
done

Reply:
done